discovery-provider/src/tasks/entity_manager/entities/track.py

```diff
@@ -114,8 +114,11 @@ def update_track_price_history(
         )
         if (
             not old_record
-            or old_record.total_price_cents != new_record.total_price_cents
-            or old_record.splits != new_record.splits
+            or old_record.block_timestamp != new_record.block_timestamp
+            and (
+                old_record.total_price_cents != new_record.total_price_cents
+                or old_record.splits != new_record.splits
```

Comment:
Is python smart enough to to a deep comparison of dicts here?

Reply:
yep, validated in a replit